README.rst

```diff
@@ -1,30 +1,12 @@
-======
-xclim
-======
+===================================
+xclim: Climate indices computations
+===================================
 
-.. image:: https://github.com/Ouranosinc/xclim/blob/master/_static/_images/xclim-logo.png
```

Comment:
No love for the logo?

Reply:
I moved it to the side-bar in RTD. I thought it occupied a lot of real estate.

Reply:
Ah, OK. I agree, it is a bit in your face. I'm just so used to seeing it, haha.

Reply:
Not closed to get it back up if you can manage to have it scaled down, possibly inline in the title.

Reply:
I could quickly look into that.

Reply:
Trying out a manually rescaled image. GitHub has eliminated any means of doing rescaling via restructuredtext unfortunately.